.github/dependabot.yml

```diff
@@ -0,0 +1,31 @@
+version: 2
+updates:
+  # Maintain Python packages
+  - package-ecosystem: "pip"
```

Comment:
pip does not apply to this repo.

Reply:
Releaser is a single script. It's not a module/package, so I don't think pip or dependabot can handle the dependencies. There is no `setup.py` or `requirements.txt`. Just https://github.com/pyTooling/Actions/blob/main/releaser/Dockerfile#L3.